### ISSUE TITLE
False negative on anchor links with SVG children

### ISSUE THREAD

**Issue by adrianengine** [dequelabs/axe-core#799]
Links with an SVG and text are triggering the following error in Accessibility:

[Deque University: Links must have discernible text](https://dequeuniversity.com/rules/axe/2.2/link-name?application=lighthouse)

The code:
```
<a href="//twitter.com" target="_blank" aria-describedby="opensNewTab" rel="noopener">
    <svg class="icon" aria-hidden="true"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-twitter"></use></svg>
    <span class="invite">Follow us on Twitter</span>
</a>
```
Testing on Voice Over the text inside the `<span>` is been read.

[Audited Website](https://www.grand79vodka.com/)

Thank you!
